api/migrations/versions/016571f41a20_add_organisation.py

```diff
@@ -0,0 +1,60 @@
+"""Add organisation model
+
+Revision ID: 016571f41a20
+Revises: 2873f4f08a4b
+Create Date: 2020-01-18 17:50:21.702261
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '016571f41a20'
+down_revision = '2873f4f08a4b'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    organisation = op.create_table('organisation',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(length=50), nullable=False),
+    sa.Column('small_logo', sa.String(length=100), nullable=False),
+    sa.Column('large_logo', sa.String(length=100), nullable=False),
+    sa.Column('domain', sa.String(length=100), nullable=False),
+    sa.Column('is_default', sa.Boolean(), nullable=False),
+    sa.PrimaryKeyConstraint('id')
+    )
+
+    op.bulk_insert(organisation,
+    [
+        {
+            'id': 1,
+            'name': 'Deep Learning Indaba',
+            'small_logo': 'logo-32x32-white.png',
+            'large_logo': 'indaba-logo-dark.png',
+            'domain': 'deeplearningindaba',
+            'is_default': True
+        },
+        {
+            'id': 2,
+            'name': 'Eastern European Machine Learning Summer School',
+            'small_logo': 'logo_eeml_white.png',
+            'large_logo': 'logo_eeml_black.png',
+            'domain': 'eeml',
+            'is_default': False
+        },
+        {
+            'id': 3,
+            'name': 'Deep Learning Indaba',
+            'small_logo': 'logo-32x32-white.png',
+            'large_logo': 'indaba-logo-dark.png',
+            'domain': 'localhost',
```

Comment:
this for testing I assume?

Reply:
Yeah, to stop the log spam during development... was going to do something with the app.yaml, but decided it's easier to just remove this record from the prod database :) Let me know if you prefer another approach though